enterprise/frontend/src/metabase-enterprise/google_drive/PausedModal.tsx

```diff
@@ -0,0 +1,86 @@
+import { t } from "ttag";
+
+import { skipToken, useGetDatabaseQuery } from "metabase/api";
+import { useSetting } from "metabase/common/hooks";
+import ExternalLink from "metabase/core/components/ExternalLink";
+import { useSelector } from "metabase/lib/redux";
+import { getUserIsAdmin } from "metabase/selectors/user";
+import { _FileUploadErrorModal } from "metabase/status/components/FileUploadStatusLarge/FileUploadErrorModal";
+import { Box, Button, Modal, Stack, Text } from "metabase/ui";
+
+import databaseError from "./database-error.svg?component";
+
+function PausedModal({ onClose }: { onClose: () => void }) {
+  const isAdmin = useSelector(getUserIsAdmin);
+
+  return (
+    <Modal opened onClose={onClose} padding="xl" withCloseButton={false}>
+      <Stack gap="md" pt="lg" ta="center">
+        <Box component={databaseError} mx="auto" />
+        <Text size="lg" fw="bold">
+          {t`Couldn't upload the file, storage is full`}
+        </Text>
+        <Text c="text-medium">
+          {isAdmin
+            ? // eslint-disable-next-line no-literal-metabase-strings -- admin only
+              t`Add more storage to your Metabase or connect a database to store the uploaded files.`
```

Comment:
I might be missing something but from the gif this looks like it might be visible to nonadmins?

Reply:
well, the first branch in the ternary is admin only, the second, that doesn't use the word "Metabase" is for non-admins